src/javascripts/modules/pills.js

```diff
@@ -1,55 +1,50 @@
 export default class Pills {
   constructor(el) {
     this.el = el
-    this.setVars()
     this.setupListeners()
   }
 
-  setVars() {
-    this.ticking = false
-    this.lastScrollY = 0
-  }
-
   setupListeners() {
     window.addEventListener('load', () => this.handleLoad())
-    window.addEventListener('scroll', () => this.handleScroll())
+    this.createObserver()
   }
 
   handleLoad() {
     this.timeout = setTimeout(() => this.animatePills(), 500)
     this.releaseSqueezeTimeout = setTimeout(() => this.releasePillSqueeze(), 3500)
   }
 
-  handleScroll() {
-    this.lastScrollY = window.scrollY
-    if (!this.ticking) {
-      window.requestAnimationFrame(() => {
-        this.shiftPills()
-      })
+  createObserver() {
+    this.observer
```

Comment:
This seems unnecessary to me, is it?

Reply:
Ah, you're right - it came from the example and I didn't realize that it's not needed.